gapis/service/service.go

```diff
@@ -190,7 +190,7 @@ type Service interface {
 
 	// ValidateDevice validates the GPU profiling capabilities of the given device and returns
 	// an error if validation failed or the GPU profiling data is invalid.
-	ValidateDevice(ctx context.Context, d *path.Device) error
+	ValidateDevice(ctx context.Context, d *path.Device) (*ValidateDeviceResponse, error)
```

Comment:
Instead of returning the proto here, could you please create a go struct here that contains the relevant validation results (at this point just the trace file name) and then pack it up into the response proto in `grpc.go`. We haven't been very consistent, but I think it's preferable to not leak the `*Request` and `*Response` protos outside `grcp.go`.

Something like:
```
type DeviceValidation struct {
  ..
}
//..
ValidateDevice(..) (*DeviceValidation, error)
```

Reply:
Done